frontend/src/components/ui/collaboration/CollabEditor.js

```diff
@@ -0,0 +1,50 @@
+import { useEffect } from 'react';
+import io from 'socket.io-client';
+import CodeMirror from 'codemirror/lib/codemirror.js';
+import 'codemirror/lib/codemirror.css'
+import 'codemirror/theme/neo.css'
+import 'codemirror/mode/javascript/javascript'
+import 'codemirror/keymap/sublime'
+import { URL_COLLAB_SVC } from '../../../configs';
```

Comment:
Similarly, spacing between the codemirror imports and configs import. Follow online convention.